### ISSUE TITLE
Dot notation differences

### ISSUE THREAD

**Issue by milesfrain** [crstnbr/JuliaWorkshop19#6]
Are these supposed to be different?
https://github.com/crstnbr/JuliaWorkshop19/blob/75635f55d93b8c690a6ac83e5b625fe779eed1d0/2_Two/3_gotchas.ipynb#L726-L727

**Comment by crstnbr** [comment]
No, I guess this was an old comment that I had in the explicit `x .= x .+ 2.*x` example before I decided to showcase the `@.` macro as well.

Thanks for pointing it out. I'll fix it soon.
